jib-core/src/main/java/com/google/cloud/tools/jib/zip/ZipUtil.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2018 Google LLC. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.zip;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+import org.apache.commons.compress.utils.IOUtils;
+
+/** Utility class for Zip archives. */
+public class ZipUtil {
```

Comment:
Will this file be used outside of the plugins? If not, we could just put it in `plugins-common`.

Reply:
Yeah, probably unzipping will happen at the plugin level, so I'll move it to `plugins-common`.